test/insight.js

```diff
@@ -0,0 +1,34 @@
+import objectValues from 'object-values';
```

Comment:
Seems like this crashes on `node.js` 4 and 6. OK if I revert to `object-values`?